client/components/push-notification/settings.jsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Dialog from 'components/dialog';
+import Gridicon from 'components/gridicon';
+import Notice from 'components/notice';
+import observe from 'lib/mixins/data-observe';
+
+const debug = debugFactory( 'calypso:push-notification-settings' );
+
+export default  React.createClass( {
+	displayName: 'PushNotificationSettings',
+
+	mixins: [ observe( 'pushNotifications' ) ],
+
+	propTypes: {
+		pushNotifications: React.PropTypes.object
+	},
+
+	getInitialState: function() {
+		return {
+			showDialog: false
+		};
+	},
+
+	clickHandler: function() {
+		debug( 'Handling click for state %s', this.props.pushNotifications.state );
+
+		if ( 'subscribed' === this.props.pushNotifications.state ) {
+			this.props.pushNotifications.unsubscribe();
+		}
+
+		if ( 'unsubscribed' === this.props.pushNotifications.state ) {
+			this.props.pushNotifications.subscribe();
+		}
+	},
+
+	getBlockedInstruction: function() {
+		return (
+			<Dialog isVisible={ this.state.showDialog } className="push-notification-instruction__dialog" onClose={ this._onCloseDialog }>
+				<div className="push-notification-instruction__content">
+					<div>
+						<div className="push-notification-instruction__title">{ this.translate( 'Enable Browser Notifications' ) }</div>
+						<div className="push-notification-instruction__step">
+							<img height="180px" width="180px" src="/calypso/images/push-notifications/address-bar.svg" />
+							<p>{ this.translate( 'Click the lock icon in your address bar.' ) }</p>
+						</div>
+						<div className="push-notification-instruction__step">
+							<img height="180px" width="180px" src="/calypso/images/push-notifications/always-allow.svg" />
+							<p>{ this.translate(
+								'Click {{strong}}Notifications{{/strong}} and choose {{em}}Always allow{{/em}}.', {
+									components: {
+										strong: <strong />,
+										em: <em />
+									} }
+							) }</p>
+						</div>
+					</div>
+				</div>
+				<span tabIndex="0" className="push-notification-instruction__dismiss" onClick={ this._onCloseDialog } >
+					<Gridicon icon="cross" size={ 24 } />
+					<span className="screen-reader-text">{ this.translate( 'Dismiss' ) }</span>
+				</span>
+			</Dialog>
+		);
+	},
+
+	_onShowDialog: function() {
+		this.setState( { showDialog: true } );
+	},
+
+	_onCloseDialog: function() {
+		this.setState( { showDialog: false } );
+	},
+
+	render: function() {
+		let blockedInstruction,
+			buttonClass,
+			buttonDisabled,
+			buttonText,
+			deniedText = null,
+			stateClass,
+			stateText;
+
+		switch ( this.props.pushNotifications.state ) {
+			case 'unsubscribed':
+				buttonClass = { 'is-enable': true };
```

Comment:
`is-enable` is for the `push-notification__settings-button` parent selector - it defines the style for the the button when it is an Enable button (i.e. pushing it will enable browser notifications).

`is-enabled` is for `push-notification__settings-state` parent selector - it defines the style for the text that describes the state of push notifications (Enabled/Disabled).